org.mixedrealitytoolkit.spatialmanipulation/Solvers/HandConstraintPalmUp.cs

```diff
@@ -294,16 +295,17 @@ private bool IsUserGazeMeetingThresholdRequirements(XRNode hand)
                     }
                 }
                 #pragma warning restore CS0618
-                else if (TrackedPoseDriverLookup != null)
+                else if (TrackedPoseDriverLookup != null &&
+                    TrackedPoseDriverLookup.GazeTrackedPoseDriver != null)
```

Comment:
Putting the check for GazeTrackedPoseDriver != null here, causes the logic to skip the fallback to headpose using camera transform.

Perhaps include a null check of the driver inside GetInputTrackingState, where it just returns InputTrackingState.None in that case, allowing the fallback.

Reply:
Ahh yes :)  I made this better.   See the new helper function for details